### ISSUE TITLE
Fallback file doesn't stop playing with Stereotool config loaded

### ISSUE THREAD

**Issue by BeyondChrispy** [AzuraCast/AzuraCast#6235]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

0.18.1

### What happened?

Have replicated this in some earlier versions too on both x86 and ARM and with versions 10 and 9.91 of Stereotool.

If no config is loaded, AutoDJ and Streamers work fine. When a Stereotool config is loaded (I tried a few), the fallback file just plays constantly despite a streamer being connected. AutoDJ doesn't play either.

Liquidsoap logs show that a DJ has connected, but the fallback continues.

**Comment by BeyondChrispy** [comment]
This is happening in AWS and Lightsail instances.  I've been unable to replicate this issue in Oracle Cloud ARM or locally on Hyper-V (all Ubuntu LTS).

**Comment by BusterNeece** [comment]
Not sure what would be causing this on our end. May require more detailed logging (if available).

**Comment by lumper5** [comment]
I have this happening as well.  I've tried a dozen .sts files - some of which I created myself, others I found in repositories.  ST and AutoDJ function as expected only when no configuration file is loaded (clearing a loaded .sts file fixes the issue).  ST can be seen (and heard) working in this "default" state but any .sts file causes endless intro.  Playing Next *is* populated, but never plays.  Let me know how best to provide relevant logs, etc.
